packaging/pom.xml

```diff
@@ -114,7 +233,7 @@
               <jre>
                 <path>%JAVA_HOME%;%PATH%</path>
                 <minVersion>1.8.0</minVersion>
-                <jdkPreference>preferJre</jdkPreference>
+                <bundledJreAsFallback>true</bundledJreAsFallback>
```

Comment:
Wait what? are we bundling the JRE now with Launch4j or no?

Reply:
No, it's a misleading option name. We do not ship the JRE, it's just that launch4j's way to inspect environment variables to look for Java relies on looking at a path containing environment variables, which is normally used to locate an embedded JRE. The launch4j maintainer is aware of this discrepancy.

Reply:
Thanks for clarifying.